pkgs/top-level/all-packages.nix

```diff
@@ -12558,6 +12558,8 @@ with pkgs;
     targetLlvmLibraries = targetPackages.llvmPackages_13.libraries;
   } // lib.optionalAttrs (stdenv.hostPlatform.isi686 && buildPackages.stdenv.cc.isGNU) {
     stdenv = gcc7Stdenv;
+  } // lib.optionalAttrs stdenv.hostPlatform.isDarwin {
+    inherit (llvmPackages_11) stdenv;
```

Comment:
This change isn't present in the original PR. It is necessary here because the Darwin stdenv in `staging-21.11` uses LLVM 7, which refuses to compile libcxx from LLVM 13.